sql/src/test/java/io/crate/integrationtests/PostgresITest.java

```diff
@@ -131,6 +132,28 @@ public void testArrayTypeSupport() throws Exception {
     }
 
     @Test
+    public void testGeoPointSupport() throws Exception {
```

Comment:
please remove the test, enabling `UseJdbc` on the existing geo_point specific integration tests should be enough